Resources/Textures/Objects/Consumable/Drinks/hot_moffee.rsi/meta.json

```diff
@@ -0,0 +1,28 @@
+{
+	"version": 1,
+	"size": {
+		"x": 32,
+		"y": 32
+	},
+	"license": "CC-BY-SA-3.0",
+	"copyright": "https://github.com/discordia-space/CEV-Eris/raw/f7aa28fd4b4d0386c3393d829681ebca526f1d2d/icons/obj/drinks.dmi",
```

Comment:
Is there any difference between this and `hot_coffee.rsi`? 

I can't seen to find a "moffee" inside of CEV-Eris, so I'm wondering if maybe you made modifications and that's why it's separated?

Reply:
Addressed in ArtisticRoomba's comment above, the vend sprite is a recolour of the hot_coffee one.